Update ml-limitations.asciidoc (#2713)
elastic/stack-docs: docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

processes is reduced to favor the {es} JVM. This improvement does not apply to 
Windows environments.

[discrete]
[[ml-license-ccs-limitations]]

> === Basic License Limitations for ML Jobs with CCS
> 
> Under a basic license, machine learning jobs cannot be initiated on datasets from remote clusters accessed through Cross-Cluster Search (CCS). To use ML jobs with CCS datasets, an upgrade to a Platinum license or higher is required.
> Refer to the [Subscriptions](https://www.elastic.co/subscriptions) page for details on features available with different subscription levels.
> 

Reviewer: ```suggestion
=== License limitations for {ml-jobs} with CCS

You must have an appropriate license to initiate {ml-jobs-cap} on datasets from remote clusters
accessed through Cross-Cluster Search (CCS). Refer to the 
[Subscriptions](https://www.elastic.co/subscriptions) page for details on features available with
different subscription levels.
```
